Gene/examples/AGR-GeneEntity-with-context_Full.json

```diff
@@ -0,0 +1,80 @@
+{
+  "@context": [
+    {
+      "bs": "http://bioschemas.org/"
+    },
+    "http://schema.org",
+    {
+      "@base": "http://schema.org"
+    }
+  ],
+  "@type": [
+    "bs:Gene"
+  ],
+  "description": "Predicted to have DNA-binding activity transcription activator activity, RNA polymerase II-specific; involved in several processes, including animal organ development, embryonic morphogenesis, and hepaticobiliary system development; predicted to localize to the nucleus.",
+  "identifier": "ZFIN:ZDB-GENE-001103-2",
+  "url":"https://www.alliancegenome.org/gene/ZFIN:ZDB-GENE-001103-2",
+  "name": "sox9b",
+  "isPartOfBioChemEntity": [
+    {
+      "@type": "BioChemEntity",
+      "name": "Chromosome 21"
+    }
+  ],
+  "hasCategoryCode": [
+    {
+      "@type": "BioChemEntity",
+      "name": "Ciliary rootlet",
+      "codeCategory": {
+        "@type": "CategoryCode",
+        "codeValue": "GO:0035253",
+        "url": "http://purl.obolibrary.org/obo/GO_0035253",
+        "inCodeSet": {
+          "@type": "CategoryCodeSet",
+          "name": "Gene Ontology"
+        }
+      }
+    }
+  ],
+  "taxonomicRange": {
+    "@type": "Taxon",
+    "name": "Danio rerio",
+    "alternateName": "Zebrafish",
+    "url": "http://purl.bioontology.org/ontology/NCBITAXON/7955",
+    "sameAs": "http://purl.uniprot.org/taxonomy/7955"
+  },
+  "hasRepresentation": "http://jbrowse.alliancegenome.org/jbrowse/index.html?data=data%2FDanio%20rerio&tracks=All%20Genes&highlight=&lookupSymbol=sox9b&loc=3:62337803..62352444",
+  "isTranscribedInto": [
+    {
+      "@type": [
+        "bs:Protein"
+      ],
+      "identifier": "uniprot:Q5DLW5",
+      "url": "https://www.uniprot.org/uniprot/Q5DLW5"
+    },
+    {
+      "@type": [
+        "bs:Protein"
+      ],
+      "identifier": "uniprot:Q6NXE1",
+      "url": "https://www.uniprot.org/uniprot/Q6NXE1"
+    },
+    {
+      "@type": [
+        "bs:Protein"
+      ],
+      "identifier": "uniprot:Q8UV13",
+      "url": "https://www.uniprot.org/uniprot/Q8UV13"
```

Comment:
Please use http://purl.uniprot.org/uniprot/ this is the protein, what you are linking to is the webpage which talks about the protein.